import.py

```diff
@@ -0,0 +1,281 @@
+  adminFilterRefs = ""
+  adminSiteRegistrations = ""
+  loadMappings = ""
+  loadPaths = ""
+  loadImports = ""
+  viewsSnuggetMatches = ""
+  templateMomentSnuggets = ""
+
+  for subdir in [reprojectedDir, simplifiedDir]:
+    if not os.path.exists(subdir):
+      os.mkdir(subdir)
+
+  i = 0
+  for f in os.listdir(dataDir):
+    if f[-4:] == ".shp":
+      stem = f[:-4].replace(".", "_").replace("-","_")
+      print("Opening shapefile:", stem)
+      reprojected = reprojectShapefile(f, dataDir, reprojectedDir, SRIDNamespace+":"+desiredSRID)
+      simplified = simplifyShapefile(reprojected, simplifiedDir, simplificationTolerance)
+      sf = shapefile.Reader(simplified)
+
+      keyField, uniqueField = askUserForFieldNames(sf, stem)
+      shapeType = detectGeometryType(sf, stem)
+      encoding = findEncoding(sf, dataDir, stem)
+
+      modelsClasses += modelClassGen(stem, sf, keyField, uniqueField, desiredSRID, shapeType)
+      modelsFilters += "    " + stem + "_filter = models.ForeignKey(" + stem
+      modelsFilters += ", related_name='+', on_delete=models.PROTECT, blank=True, null=True)\n"
+      modelsGeoFilters += modelsGeoFilterGen(stem, keyField)
+      modelsSnuggetGroups += "                          '"
+      modelsSnuggetGroups += stem + "_snugs': " + stem + "_snuggets,\n"
+      modelsSnuggetRatings += "                '"
+      modelsSnuggetRatings += stem + "_rating': " + stem + "_rating,\n"
+
+      adminModelImports += ", " + stem
+      if i > 0:
+        adminFilterRefs += ", "
+      adminFilterRefs += "'" + stem + "_filter'"
+      adminSiteRegistrations += "admin.site.register(" + stem
+      adminSiteRegistrations += ", GeoNoEditAdmin)\n"
+
+      loadMappings += stem + "_mapping = {\n"
+      loadMappings += "    '" + keyField.lower() + "': '" + keyField + "',\n"
+      if keyField != uniqueField:
+        loadMappings += "    '" + uniqueField.lower() + "': '" + uniqueField + "',\n"
+      loadMappings += "    'geom': '" + shapeType.upper() + "'\n"
+      loadMappings += "}\n\n"
+      loadPaths += stem + "_shp = " + "os.path.abspath(os.path.join(os.path.dirname(__file__),"
+      loadPaths += " '../" + simplified + "'))\n"
+      loadImports += "    from .models import " + stem + "\n"
+      loadImports += "    lm_" + str(i) + " = LayerMapping(" + stem + ", "
+      loadImports += stem + "_shp, " + stem + "_mapping, transform=True, "
+      loadImports += "encoding='" + encoding
+      loadImports += "', unique=['" + uniqueField.lower() + "'])\n"
+      loadImports += "    lm_" + str(i) + ".save(strict=True, verbose=verbose)\n\n"
+
+      viewsSnuggetMatches += "            if snugget_content['structured']['moment']['"
+      viewsSnuggetMatches += stem + "_snugs']:\n"
+      viewsSnuggetMatches += "                wrapper_width += base_section_width\n"
+      viewsSnuggetMatches += "                n_sections += 1\n"
+
+      print("")
+      i = i + 1
```

Comment:
if you're not using this as a salt, it would be better to make it a boolean flag.